dwitter/templates/snippets/remix_info.html

```diff
@@ -1,13 +1,13 @@
 {% load to_gravatar_url %}
 <div class="remix-info">
   remix of
-  <a href="{% url 'dweet_show' dweet_id=dweet.reply_to.id %}">#{{dweet.reply_to.id}}</a>
-  by
   {% if dweet.reply_to.deleted %}
     [deleted]
   {% else %}
+    <a href="{% url 'dweet_show' dweet_id=dweet.reply_to.id %}">d/{{dweet.reply_to.id}}</a>
```

Comment:
Could you put the d/ and u/ in a `<span class="faded"></span>` (`.faded { opacity: .75 }`)?